README.md

```diff
@@ -27,10 +27,10 @@ Kubernetes 1.16+ is the minimum recommended version for KFServing.
 
 Knative Serving and Istio should be available on Kubernetes Cluster, KFServing currently depends on Istio Ingress Gateway to route requests to inference services.
 
-- [Istio](https://knative.dev/docs/install/installing-istio): v1.3.1+
+- [Istio](https://knative.dev/docs/install/installing-istio): v1.9.0+
```

Comment:
@animeshsingh  I think KFServing can follow the istio [supported versions](https://istio.io/latest/about/supported-releases/#support-status-of-istio-releases)

Reply:
and who is going to test all the metrix there? Given KF is being moved to 1.9+, there is no reason to hold back